### ISSUE TITLE
Cypress 7 support?

### ISSUE THREAD

**Issue by alejandroiglesias** [bahmutov/cy-rollup#162]
Hi, I have been using this library for a while and I'm wondering if there's planned support for Cypress 7. In case the library already supports using Cypress 7, can you please instruct and/or update the readme to know how to use it with that newer version?
Thanks and I appreciate your help.

**Comment by bahmutov** [comment]
I can see in this repo's README that it use Cypress v7 already https://github.com/bahmutov/cy-rollup#bahmutovcy-rollup-----

Is it not clear that it supports Cypress 7?

**Comment by alejandroiglesias** [comment]
It is not clear because the README shows initialization like this:
```js
on('file:preprocessor', rollupPreprocessor(options))
```
While in the [Cypress 7 Migration Guide](https://docs.cypress.io/guides/references/migration-guide#Component-Testing) it's stated that component testing should be configured in a different way through
```js
on('dev-server:start', (options) => {
  return startDevServer({ options, webpackConfig })
})
```
Can you please provide more info?

**Comment by gorjan-mishevski** [comment]
This plugin is broken for Cypress 7 and up. Nor does the cypress component lib works. Dead repo

**Comment by bahmutov** [comment]
Right, this plugin so far only supports end-to-end tests, not the component test running approach Cypress v7 has introduced. I welcome pull requests!
